src/lib/operators/table_scan.cpp

```diff
@@ -19,6 +19,7 @@
 #include "storage/reference_column.hpp"
 #include "storage/table.hpp"
 #include "table_scan/column_comparison_table_scan_impl.hpp"
+#include "table_scan/empty_table_scan_impl.hpp"
```

Comment:
I don't like "empty" here much. What is empty? The input? The output? How about `always_(true/false)_table_scan`?

Reply:
I think I found a better solution that does not involve creating an empty table scan implementation class.

See 99da271be5205e4c8dc1dca0964e7ae0062e7f82